metron-interface/metron-rest/src/main/java/org/apache/metron/rest/controller/GrokController.java

```diff
@@ -53,4 +54,11 @@
     ResponseEntity<Map<String, String>> list() throws RestException {
         return new ResponseEntity<>(grokService.getCommonGrokPatterns(), HttpStatus.OK);
     }
+
+    @ApiOperation(value = "Retrieves a Grok statement from the classpath")
+    @ApiResponse(message = "Grok statement", code = 200)
+    @RequestMapping(value = "/get/statement", method = RequestMethod.GET)
```

Comment:
Getting the grokStatement from HDFS is a different REST endpoint.  The logic does mimic GrokParser.openInputStream, it's just on the client side.